### PR TITLE
Add tested model modem and solution for RPI

### PR DESCRIPTION
**Pull request by renat85**
Add tested model modem and solution for work on Raspberry PI without udev rule.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by renat85** [comment]
Add tested model modem and solution for work on Raspberry PI without udev rule.

File: source/_integrations/sms.markdown [hunk 1]
```diff
@@ -77,13 +77,16 @@ You will need a USB GSM stick modem.
...
 - [Huawei E3531](https://www.amazon.com/Modem-Huawei-Unlocked-Caribbean-Desbloqueado/dp/B011YZZ6Q2/ref=sr_1_1?keywords=Huawei+E3531&qid=1581447800&sr=8-1)
+- [Huawei E3272](https://www.amazon.com/Huawei-E3272s-506-Unlocked-Americas-Europe/dp/B00HBL51OQ)
 
 [List of modems that may work](https://www.asus.com/event/networks_3G4G_support/)
 
 ### Huawei modems on Raspberry Pi (and similar) devices
 
 For some unknown reason, the rule that converts these modems from storage devices into serial devices does not run automatically. To work around this problem, follow the procedure to create `udev` rule on a configuration USB stick for the device to switch to serial mode.
 
+0. Try disable virtual cd-rom and change work mode "only modem". After this modem correct work on Raspberry Pi without 'udev' rule.
```
Comment (renat85) [inline source/_integrations/sms.markdown:88]: For first not need create udev rule. Шncorrect definition is associated with virtual cdrom. Just need disable this.

File: source/_integrations/sms.markdown [hunk 2]
```diff
@@ -77,13 +77,16 @@ You will need a USB GSM stick modem.
 - [Huawei E3372-510](https://www.amazon.com/gp/product/B01N6P3HI2/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1)(
 Need to unlock it using [this guide](http://blog.asiantuntijakaveri.fi/2015/07/convert-huawei-e3372h-153-from.html))
 - [Huawei E3531](https://www.amazon.com/Modem-Huawei-Unlocked-Caribbean-Desbloqueado/dp/B011YZZ6Q2/ref=sr_1_1?keywords=Huawei+E3531&qid=1581447800&sr=8-1)
+- [Huawei E3272](https://www.amazon.com/Huawei-E3272s-506-Unlocked-Americas-Europe/dp/B00HBL51OQ)
```
Comment (renat85) [inline source/_integrations/sms.markdown:80]: New model which I tested in my HA.

Comment (renat85) [inline source/_integrations/sms.markdown:80]: Added tested modem model in my HA

File: source/_integrations/sms.markdown (hunk 1 above)
Comment (renat85) [inline source/_integrations/sms.markdown:88]: Its solution without create udev rule

**Review by frenck — APPROVED** [review]
Thanks, @renat85 👍

**Pull request merged**